src/main/java/com/google/gerrit/extensions/common/EditInfo.java

```diff
@@ -18,7 +18,7 @@
 
 public class EditInfo {
   public CommitInfo commit;
-  public String baseRevision;
+  public String base_revision;
```

Comment:
I don't think this change is required? It should convert fine now also with camel case notation?